ioctl/newcmd/account/accountactions_test.go

```diff
@@ -104,10 +101,8 @@ func TestNewAccountAction(t *testing.T) {
 		client.EXPECT().QueryAnalyser(gomock.Any()).DoAndReturn(func(reqData interface{}) (*http.Response, error) {
```

Comment:
one question: why do we need this mock?

Reply:
`QueryAnalyser`  is added into Client interface by https://github.com/iotexproject/iotex-core/pull/3269.